.gitignore

```diff
@@ -34,3 +34,5 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+package-lock.json
```

Comment:
Do not ignore `package-lock.json`. Also make sure that versions defined in `package-lock.json` match those in `yarn.lock`, so that regardless of package manger used to install dependencies, they always resolve to same version.

Reply:
The reasoning here is that the backend-repo is configured to use node 4.8.4 and npm 2.15.11 so there should be no package-lock at all, but sometimes .nvmrc is ignored and (like on Windows) and one is generated, because the adminpanel uses a newer version of node and npm

Reply:
Fair enough 👍